analyzer/queries/queries.go

```diff
@@ -808,4 +834,104 @@ var (
       source_files = $5
     WHERE
       runtime = $1 AND contract_address = $2`
+
+	RuntimeEvmVerifiedContractTxs = `
+    WITH abi_contracts AS (
+      SELECT 
+        runtime,
+        contract_address AS addr,
+        abi
+      FROM chain.evm_contracts
+      WHERE
+        runtime = $1 AND verification_info_downloaded_at IS NOT NULL
```

Comment:
```suggestion
        runtime = $1 AND abi IS NOT NULL
```
Nit. This is a little more explicit and a little safer (because we're not relying on the fact that other code will only ever set `downloaded_at` when they also set `abi`)